src/client/game/systems/ParticlePool.ts

```
@@ -0,0 +1,302 @@
+     */
+    public emitBoost(x: number, y: number, z: number): void {
+        this.emit(x, y, z, { type: 'BOOST' });
+    }
+
+    /**
+     * Update all active particles. Called each frame.
+     */
+    public update(dt: number): void {
+        for (let i = 0; i < this.maxParticles; i++) {
+            const particle = this.particles[i];
+            if (!particle.active) continue;
```

Biome (2.4.4)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)